src/openms/source/TRANSFORMATIONS/FEATUREFINDER/EGHTraceFitter.cpp

```diff
@@ -279,7 +279,8 @@ namespace OpenMS
   {
     // equation 21 from Lan & Jorgenson paper:
     double abs_tau = fabs(tau_);
-    double phi = atan(abs_tau / sigma_);
+    double abs_sigma = fabs(sigma_);
+    double phi = atan(abs_tau / abs_sigma);
```

Comment:
how can sigma ever by negative? This is supposed to be the square root of the variance ... if this is negative, then the fit failed, right? I think simply setting it to the absolute value is a hack but leads to very wrong results ... should it not just be zero then? 

Also, do we have an example of such a degenerate case? maybe we can add it to the tests?